projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+type RequestError struct {
+	SleepTime int
+	Retry     bool
```

Comment:
Maybe `Retriable` or `ShouldRetry`? Named just `Retry` isn't no so clear whether this is saying "you should retry" or "this was a retry".